src/components/settings/SpendingLimits/SpendingLimitsTable.tsx

```diff
@@ -47,7 +47,10 @@ export const SpendingLimitsTable = ({ spendingLimits }: { spendingLimits: Spendi
       spendingLimits.map((spendingLimit) => {
         const token = balances.items.find((item) => item.tokenInfo.address === spendingLimit.token)
         const amount = BigNumber.from(spendingLimit.amount)
-        const formattedAmount = formatVisualAmount(amount, token?.tokenInfo.decimals)
+        const formattedAmount = safeFormatUnits(amount, token?.tokenInfo.decimals)
+
+        const spent = BigNumber.from(spendingLimit.spent)
+        const formattedSpent = safeFormatUnits(spent, token?.tokenInfo.decimals)
```

Comment:
Now the other PR is merged, should these not be `formatVisualAmount`?

```suggestion
        const formattedAmount = safeFormatUnits(amount, token?.tokenInfo.decimals)

        const spent = BigNumber.from(spendingLimit.spent)
        const formattedSpent = safeFormatUnits(spent, token?.tokenInfo.decimals)
```

Reply:
I replied in the other comment but I think `safeFormatUnits` makes more sense here because we display 2 values next to each other. With `formatVisualAmount` the values could be formatted differently like `< 0.00001 of 0.09 ETH`  instead of `0.000002 of 0.09 ETH`